js/create-index.js

```diff
@@ -0,0 +1,33 @@
+var lunr = require('lunr')
+var fs = require('fs');
+var data;
+
+buildIndex();
+
+function buildIndex() {
+  fs.readFile('_site/search_data.json', 'utf8', function(err, data) {
```

Comment:
I think you'll want to use some method that's a little more robust in terms of handling relative paths and being run from multiple places. Look at `__filename` and `__dirname`: https://stackoverflow.com/questions/3133243/how-do-i-get-the-path-to-the-current-script-with-node-js

Reply:
Not sure if my fix is what you were getting at, but I think it handles the situation.